index.js

```diff
@@ -56,13 +57,20 @@ const init = async () => {
   // Then we load events, which will include our message and ready event.
   const evtFiles = await readdir("./events/");
   client.logger.log(`Loading a total of ${evtFiles.length} events.`);
-  evtFiles.forEach(file => {
+  await Promise.all(evtFiles.map(async (file) => {
     const eventName = file.split(".")[0];
     const event = require(`./events/${file}`);
     // This line is awesome by the way. Just sayin'.
     client.on(eventName, event.bind(null, client));
     delete require.cache[require.resolve(`./events/${file}`)];
-  });
+  }));
+
+  const backgroundFiles = await readdir("./background/");
+  client.logger.log(`Loading a total of ${backgroundFiles.length} background workers.`);
+  await Promise.all(backgroundFiles.map(async (file) => {
```

Comment:
This does not need to await async functions either.

Reply:
See above